doc/nrf/device_guides/working_with_nrf/nrf53/qspi_xip_guide.rst

```diff
@@ -0,0 +1,324 @@
+.. _qspi_xip:
+
+Configuring external XIP
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes the external XIP support for the nRF5340 SoC.
+The nRF5340 SoC is equipped with QSPI memory interface which is capable of exposing QSPI flash as memory for CPU used to fetch and execute the program code.
+The QSPI is available for the application core.
+Therefore, it is possible to relocate a part of the application's code to an external memory.
+NCS supports dividing an application into an internal and external part, along with the MCUboot support.
+
+Project setup
```

Comment:
Hm, I know this is taken mostly from the Word draft that was provided, but perhaps the following subheadings can be made on the same level as this one?

In that case, maybe the title would need to be changed to something else... perhaps "Overview", just a thought.

Reply:
Sounds/looks good!